doc/user_manual/postprocessor.tex

```diff
@@ -1473,6 +1473,41 @@ \subsubsection{Interfaced}
 </Simulation>
 \end{lstlisting}
 
+\paragraph{Method: HistorySetDelay}
+
+This Post-Processor allows history sets to add delayed or lagged
+variables.  For example, if there a variable price that is set hourly,
+than new variable called price\_prev\_hour could be set by using a
+delay of -1.  This can be useful for training a ROM or other data
+analysis.
+
+In the \xmlNode{PostProcessor} input block, one or more of the following XML sub-nodes are required:
+
+\begin{itemize}
+\item \xmlNode{delay}, \xmlDesc{empty}, a delay node with the following required parameters:
+  \begin{itemize}
+  \item \xmlAttr{original}, the variable to start with
```

Comment:
This is not the format of the manual.
Example:

 ```\item \xmlAttr{VAR}, \xmlDesc{required string attribute}, user-defined VARIABLE name etc etc```

Reply:
Fixed.